packages/modal/src/Modal.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { Fragment, useCallback, useRef, SetStateAction } from 'react';
+import PropTypes from 'prop-types';
+import Portal from '@leafygreen-ui/portal';
+import Icon, { Size } from '@leafygreen-ui/icon';
+import { useEventListener } from '@leafygreen-ui/hooks';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+export const ModalSize = {
+  XXSmall: 'xxsmall',
+  XSmaill: 'xsmall',
+  Small: 'small',
+  Normal: 'normal',
+  Large: 'large',
+  XLarge: 'xlarge',
+} as const;
+
+export type ModalSize = typeof ModalSize[keyof typeof ModalSize];
+
+const overlayStyle = css`
+  animation: fade-in 500ms cubic-bezier(0.165, 0.84, 0.44, 1);
+  -webkit-animation: fade-in 500ms cubic-bezier(0.165, 0.84, 0.44, 1);
+  background-color: #000;
+  opacity: 0.5;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const modalContentStyle = css`
+  margin: 18px 40px;
+  max-width: 1270px;
+  min-width: 944px;
+  padding: 30px;
+  align-self: flex-start;
+  text-align: left;
```

Comment:
This should be the default for the property.

Reply:
👍